scipy/stats/tests/test_hypotests.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import (assert_, assert_equal, assert_almost_equal,
+                           assert_array_almost_equal, assert_allclose)
+
+from pytest import raises as assert_raises
+from scipy.stats._hypotests import epps_singleton_2samp, cvm_test, _cdf_cvm
+from scipy.stats import distributions
+from .common_tests import check_named_results
+
+
+class TestEppsSingleton(object):
+    def test_statistic_1(self):
+        # first example in Goerg & Kaiser, also in original paper of
+        # Epps & Singleton. Note: values do not match exactly, the
+        # value of the interquartile range varies depending on how
+        # quantiles are computed
+        x = np.array([-0.35, 2.55, 1.73, 0.73, 0.35, 2.69, 0.46, -0.94, -0.37, 12.07])
+        y = np.array([-1.15, -0.15, 2.48, 3.25, 3.71, 4.29, 5.00, 7.74, 8.38, 8.60])
```

Comment:
```suggestion
        x = np.array([-0.35, 2.55, 1.73, 0.73, 0.35, 
                      2.69, 0.46, -0.94, -0.37, 12.07])
        y = np.array([-1.15, -0.15, 2.48, 3.25, 3.71, 
                      4.29, 5.00, 7.74, 8.38, 8.60])
```
Due to stricter linting rules.